clang/lib/CodeGen/Targets/Sparc.cpp

```diff
@@ -325,14 +325,19 @@ Address SparcV9ABIInfo::EmitVAArg(CodeGenFunction &CGF, Address VAListAddr,
     break;
 
   case ABIArgInfo::Ignore:
-    return Address(llvm::UndefValue::get(ArgPtrTy), ArgTy, TypeInfo.Align);
+    return CGF.EmitLoadOfAnyValue(
+        CGF.MakeAddrLValue(
```

Comment:
This looks very wrong... but I guess that's not related to this patch.

Reply:
Yeah, we should probably separately add some function to synthesize an ignored value.

Reply:
I think this should be done in a separate PR.

Reply:
If you're going to switch all the other Ignore handlers to just `return Slot.asRValue();`, might as well also change this one.

Reply:
Sure, done.